homepage/_packages/cloud2edge/tour.md

```diff
@@ -19,38 +19,66 @@ or that you might experience some issues when testing.
 set as environment variables so that all commands can easily make use
-of those variables:
+of those variables. Note that you will need to set the environment variables
+in every command shell separately.
 
 {% variants %}
-{% variant Minikube %}
-{% clipboard %}
-DEVICE_REGISTRY_URL=$(minikube service -n cloud2edge c2e-service-device-registry-ext --url --https | grep 31443)
-MQTT_ADAPTER_HOST=$(minikube service -n cloud2edge c2e-adapter-mqtt-vertx --url | grep 30883 | sed -e 's/^http:\/\///' -e 's/:.*$//')
-MQTT_ADAPTER_PORT=30883
-{% endclipboard %}
-{% endvariant %}
 
 {% variant Kubernetes %}
+<br>
+Download the [setCloud2EdgeEnv script](../scripts/setCloud2EdgeEnv.sh) and use it
+to set environment variables for the Cloud2Edge package's service endpoints.
+
 {% clipboard %}
-DEVICE_REGISTRY_URL=https://$(kubectl get service c2e-service-device-registry-ext --output='jsonpath={.status.loadBalancer.ingress[0].ip}' -n cloud2edge):28080
-MQTT_ADAPTER_HOST=$(kubectl get service c2e-adapter-mqtt-vertx --output='jsonpath={.status.loadBalancer.ingress[0].ip}' -n cloud2edge)
-MQTT_ADAPTER_PORT=8883
+RELEASE=c2e
+NS=cloud2edge
+./setCloud2EdgeEnv.sh $RELEASE $NS
 {% endclipboard %}
 {% endvariant %}
 
 {% variant OpenShift %}
 {% clipboard %}
-DEVICE_REGISTRY_URL=https://$(oc get -n cloud2edge route c2e-service-device-registry-https --template='{{"{{.spec.host"}}}}')
-MQTT_ADAPTER_URL=$(oc -n cloud2edge get route c2e-adapter-http-vertx-sec --template='{{"{{.spec.host"}}}}')
-MQTT_ADAPTER_PORT=443
+NS=cloud2edge
+RELEASE=c2e
+DEVICE_REGISTRY_URL=https://$(oc get -n $NS route ${RELEASE}-service-device-registry-https --template='{{"{{.spec.host"}}}}')
+HTTP_ADAPTER_URL=$(oc get -n $NS route ${RELEASE}-adapter-http-vertx-sec --template='{{"{{.spec.host"}}}}')
+HTTP_ADAPTER_PORT=443
 {% endclipboard %}
 {% endvariant %}
 
 {% endvariants %}
 
+## Publishing telemetry data
+
+The Cloud2Edge package comes with a pre-provisioned example device that we will use to publish
+some telemetry data via Hono's HTTP protocol adapter. The data will automatically be forwarded to Ditto
+where the device's twin representation will be updated with the data published by the device.
+
+The demo device's digital twin supports a temperature property which will be set to 45
+by means of the following command:
+
+{% clipboard %}
+curl -i -u demo-device@org.eclipse.packages.c2e:demo-secret -H "application/json" --data-binary '{
+  "topic": "org.eclipse.packages.c2e/demo-device/things/twin/commands/modify",
+  "headers": {},
+  "path": "/features/temperature/properties/value",
+  "value": "45"
+}' http://${HTTP_ADAPTER_IP}:${HTTP_ADAPTER_PORT_HTTP}/telemetry
+{% endclipboard %}
+
+## Retrieving the digital twin's current state
+
+The updated state of the digital twin can then be retrieved using:
+
+{% clipboard %}
+curl -u ditto:ditto http://$DITTO_API_IP:$DITTO_API_PORT_HTTP/api/2/things/org.eclipse.packages.c2e:demo-device
+{% endclipboard %}
+
```

Comment:
Additionally to the curl polling we could also add subscribing to the Ditto "SSE" (Server Sent Events) which gets each modification to the twin pushed:
```
{% clipboard %}
curl --http2 -u ditto:ditto -H "Accept:text/event-stream" -N http://$DITTO_API_IP:$DITTO_API_PORT_HTTP/api/2/things
{% endclipboard %}
```

Reply:
That's even better, great idea :+1: